Instruments/bridge12.py

```diff
@@ -482,11 +479,15 @@ def zoom(self, dBm_increment=2, n_freq_steps=15):
         # with the new time constant added for freq_sweep, should we eliminate fast_run?
         rx, tx = self.freq_sweep(freq, fast_run=True)
         # }}}
+        self.freq_bounds = r_[start_f,stop_f]
         # MISSING -- DO BEFORE MOVING TO HIGHER POWERS!
         # test to see if any of the powers actually exceed the safety limit
         # if they do, then contract freq_bounds to include those powers
         min_f = freq[rx.argmin()]
-        return rx, tx, min_f
+        if abs(center - min_f)>0.2e6:
+            center = min_f
```

Comment:
say something here (print, I think)

Reply:
no, it should be more informative -- you're saying that the min measured value and the bottom of the parabola are too far apart, so you should have some type of warning